license.txt

```diff
@@ -0,0 +1,3 @@
+cc0 
```

Comment:
no certain of the relevane of this here.

Reply:
this is the license for the .yaml file 
its probably insignificant now that the license is specified in the appdata.xml  (or however that is called )

Reply:
it's like putting a post it note "it's mine"

Reply:
no its cc0 (puplic domain basically ) so its like a post it note with the text do whatever you want with it